centaur/src/main/resources/integrationTestCases/germline/joint-discovery-gatk/joint-discovery-gatk4-part2.wdl

```diff
@@ -0,0 +1,998 @@
+## recalibration. This would lead to a longer wall clock time for complete workflow
+## execution. Wiring the INDEL recalibration to be applied first solves the problem.
+##
+## Cromwell version support
+## - Successfully tested on v29
+## - Does not work on versions < v23 due to output syntax
+##
+## Runtime parameters are optimized for Broad's Google Cloud Platform implementation.
+## For program versions, see docker containers.
+##
+## LICENSING :
+## This script is released under the WDL source code license (BSD-3) (see LICENSE in
+## https://github.com/broadinstitute/wdl). Note however that the programs it calls may
+## be subject to different licenses. Users are responsible for checking that they are
+## authorized to run all programs before running this script. Please see the docker
+## page at https://hub.docker.com/r/broadinstitute/genomes-in-the-cloud/ for detailed
+## licensing information pertaining to the included programs.
+
+workflow JointGenotyping {
+  File sitesonlygather_output_vcf
+  File sitesonlygather_output_vcf_index
+  #HardFilter
+  Array[File] hardfilter_variant_filtered_vcf
+  Array[File] hardfilter_variant_filtered_vcf_index
+  Array[File] hardfilter_sites_only_vcf
+  Array[File] hardfilter_sites_only_vcf_index
+  File unpadded_intervals_file
+
+  String callset_name
+  File sample_name_map
+
+  File ref_fasta
+  File ref_fasta_index
+  File ref_dict
+
+  File dbsnp_vcf
+  File dbsnp_vcf_index
+
+  String gatk_docker
+  String gatk_path
+  String python_docker
+
+  Int small_disk
+  Int medium_disk
+  Int huge_disk
+
+  Array[String] snp_recalibration_tranche_values
+  Array[String] snp_recalibration_annotation_values
+  Array[String] indel_recalibration_tranche_values
+  Array[String] indel_recalibration_annotation_values
+
+  File eval_interval_list
+  File hapmap_resource_vcf
+  File hapmap_resource_vcf_index
+  File omni_resource_vcf
+  File omni_resource_vcf_index
+  File one_thousand_genomes_resource_vcf
+  File one_thousand_genomes_resource_vcf_index
+  File mills_resource_vcf
+  File mills_resource_vcf_index
+  File axiomPoly_resource_vcf
+  File axiomPoly_resource_vcf_index
+  File dbsnp_resource_vcf = dbsnp_vcf
+  File dbsnp_resource_vcf_index = dbsnp_vcf_index
+
+  # ExcessHet is a phred-scaled p-value. We want a cutoff of anything more extreme
+  # than a z-score of -4.5 which is a p-value of 3.4e-06, which phred-scaled is 54.69
+  Float excess_het_threshold = 54.69
+  Float snp_filter_level
+  Float indel_filter_level
+  Int SNP_VQSR_downsampleFactor
+
+  Int num_of_original_intervals = length(read_lines(unpadded_intervals_file))
+  Int num_gvcfs = length(read_lines(sample_name_map))
+
+  # Make a 2.5:1 interval number to samples in callset ratio interval list
+  Int possible_merge_count = floor(num_of_original_intervals / num_gvcfs / 2.5)
+  Int merge_count = if possible_merge_count > 1 then possible_merge_count else 1
+
+#  call DynamicallyCombineIntervals {
```

Comment:
same comment as Dan made above about commented out code

Reply:
I'll remove the wdl, full reponse on Dan's comment